.github/workflows/default.yml

```diff
@@ -0,0 +1,88 @@
+name: Mimiker
+
+on: [push]
+
+jobs:
+  verify_formatting:
+    name: Verify formatting
+    runs-on: self-hosted
+    container: cahirwpz/mimiker-circleci:1.7.3
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./verify-format.sh
+
+  verify_pycodestyle:
+    name: Verify pycodestyle
+    runs-on: self-hosted
+    container: cahirwpz/mimiker-circleci:1.7.3
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./verify-pycodestyle.sh
+
+  build_mips_gcc:
+    name: Build MIPS - GCC with KASAN
+    runs-on: self-hosted
+    container: cahirwpz/mimiker-circleci:1.7.3
+    steps:
+      - uses: actions/checkout@v2
+      - run: make -j $(nproc) BOARD=malta KASAN=1
+      - uses: actions/upload-artifact@v2
+        with:
+          name: "mips gcc kernel+ramdisk"
+          path: |
+            sys/mimiker.elf
+            sys/mimiker.elf.map
+            initrd.cpio
+
+  build_mips_clang:
+    name: Build MIPS - Clang
+    runs-on: self-hosted
+    container: cahirwpz/mimiker-circleci:1.7.3
+    steps:
+      - uses: actions/checkout@v2
+      - run: make -j $(nproc) BOARD=malta CLANG=1
+      - uses: actions/upload-artifact@v2
+        with:
+          name: "mips clang kernel+ramdisk"
+          path: |
+            sys/mimiker.elf
+            sys/mimiker.elf.map
+            initrd.cpio
+
+  build_aarch64_gcc:
+    name: Build AArch64 - GCC
+    runs-on: self-hosted
+    container: cahirwpz/mimiker-circleci:1.7.3
+    steps:
+      - uses: actions/checkout@v2
+      - run: make -j $(nproc) BOARD=rpi3
+
+  build_aarch64_clang:
+    name: Build AArch64 - Clang
+    runs-on: self-hosted
+    container: cahirwpz/mimiker-circleci:1.7.3
+    steps:
+      - uses: actions/checkout@v2
+      - run: make -j $(nproc) BOARD=rpi3 CLANG=1
+
+  kernel_tests_mips_gcc:
+    name: Tests MIPS - GCC with KASAN
+    runs-on: self-hosted
+    container: cahirwpz/mimiker-circleci:1.7.3
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/download-artifact@v2
```

Comment:
You have an implicit dependency that fails if a runner doesn't pick up `build_mips_gcc` earlier.

Reply:
Done.